src/main/java/appeng/integration/IntegrationType.java

```diff
@@ -36,13 +36,23 @@ public IIntegrationModule createInstance()
 		}
 	},
 
-	RC( IntegrationSide.BOTH, "Railcraft", "Railcraft" ),
+	RF( IntegrationSide.BOTH, "RedstoneFlux Power - Tiles", "CoFHAPI" ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)